neps/nep-0568.md

```diff
@@ -155,11 +153,17 @@ This allows child shards to access and update state data under the ancestor shar
 Initially, `ShardUIdMapping` is empty, as existing shards map to themselves. During resharding, a mapping entry is added to `ShardUIdMapping`,
 pointing each child shard’s `ShardUId` to the appropriate ancestor. Mappings persist as long as any descendant shard references the ancestor’s data.
 Once a node stops tracking all children and descendants of a shard, the entry for that shard can be removed, allowing its data to be garbage collected.
-For archival nodes, mappings are retained indefinitely to maintain access to the full historical state.
 
 This mapping strategy enables efficient shard management during resharding events,
 supporting smooth transitions without altering storage structures directly.
 
+#### Integration with cold storage (archival nodes)
+Cold storage uses the same mapping strategy to manage shard state during resharding:
+* When state data is migrated from hot to cold storage, it retains the parent shard’s `ShardUId` prefix, ensuring consistency with the mapping strategy.
+* While copying data for the final block of the epoch where resharding occured, the `DBCol::StateShardUIdMapping` column is copied into cold storage. This ensures that mappings are updated alongside the shard state data.
+* These mappings are permanent in cold storage, aligning with its role in preserving historical state.
+
+This approach minimizes complexity while maintaining consistency across hot and cold storage.
```

Comment:
Can you also mention that the mapping in cold storage is permanent?

Reply:
In line 164 there is "These mappings are permanent in cold storage".